docs/pul_request_template.md

```diff
@@ -0,0 +1,15 @@
+### Ticket
```

Comment:
the file name is missing an l (pul-> pull)

Reply:
I would perhaps use the path .github/pull_request_template.md to make it hidden (and we already have this .github folder)